python/hidet/graph/ops/normalize/resolve.py

```diff
@@ -32,15 +29,6 @@ class NormalizeResolveRule(ResolveRule):
     2) resolve_generic: Default case, return the output of the regular f32 reduce schedule.
     """
 
-    def resolve_f16(self, op: Operator) -> Optional[List[Tensor]]:
-        dims = op.attrs['dims']
-        x: Tensor = op.inputs[0]
-        if not is_contiguous_norm(dims, len(x.shape)):
-            return None
-        if x.dtype != dtypes.float16 or prod([x.shape[dd] for dd in dims]) % 2 != 0:
```

Comment:
this is safe for now, but we might need to think about how to handle it when we decide to use 2xfp16 types and the norm size is odd.